libr/core/cmd_print.c

```diff
@@ -5590,7 +5590,7 @@ l = use_blocksize;
 						core->num->value = r_core_print_disasm (core->print,
 							core, addr, block1, addrbytes * l, l, 0, 1, formatted_json, NULL, NULL);
 					} else {
-						eprintf ("Cannot allocate %d byte(s)\n", addrbytes * l);
+						eprintf ("Cannot allocate %ld byte(s)\n", addrbytes * l);
```

Comment:
as i said, addrbytes is int in r_io, i would probably go for casting (int) instead of changing this format, the value of addrbytes shouldnt be larger than 8